index.ts

```diff
@@ -0,0 +1,19 @@
+function fizzBuzz(n: number) {
+    for(let i = 1;i <= n;i++){
+        const strings = []; // Using a array of strings to save the different parts of the answer
+        //Decided to do this because we can't change a const and I didn't want to keep track of which string we have to print at the end
```

Comment:
You will want to leave a space between // and the comment

Generally it's best to put the comments on the line before the commented on line